src/main/fc/cli.c

```diff
@@ -402,12 +402,18 @@ static uint16_t getValueOffset(const clivalue_t *value)
     return 0;
 }
 
-STATIC_UNIT_TESTED void *getValuePointer(const clivalue_t *value)
+void *cliGetValuePointer(const clivalue_t *value)
 {
     const pgRegistry_t* rec = pgFind(value->pgn);
     return CONST_CAST(void *, rec->address + getValueOffset(value));
```

Comment:
(BTW: what is `CONST_CAST`? It's name makes no sense here ...

Reply:
It's analogous to <const_cast> in C++

Reply:
Sorry, getting too old for new c++ features ...

Reply:
I'm more of a C++ programmer than a C programmer.